jenkins_jobs/trigger_host_os_iso_build_from_versions_repo/script.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+set -e
+set -x
+
+
+TARGET_CONTEXT="Build Host OS"
+
+
+get_build_state(){
+    eval $(github_api "$GITHUB_USER_NAME" "$GITHUB_PASSWORD" \
+                      query_status "${GITHUB_ORGANIZATION_NAME}/versions" \
+                      "$ghprbPullId" "$TARGET_CONTEXT" --state \
+                      --target-url || echo "exit 1")
```

Comment:
The echo exists only because of the 'eval'. There is nothing that the script could print (aside from "exit 1") that would make the subshell exit. Moreover, this is only force it to exit right after the failed command (github_api) and not in the next command that breaks (probably the eval itself, since I'm using set -e)